packages/core/src/interfaces/common/utils/range-store.interface.ts

```diff
@@ -0,0 +1,11 @@
+
+import BigNum = require('bn.js')
+
+export interface Range {
+  start: BigNum
+  end: BigNum
+}
```

Comment:
You can import a range interface here -- https://github.com/plasma-group/pigi/blob/a1611bd23b13f9d70d246774a4900b36987ecfd6/packages/utils/src/interfaces/data-types.ts

Also we need to reintroduce these data-types which got deleted & were reintroduced here: https://github.com/karlfloersch/pigi/tree/master/packages/utils/src/data-types

Reply:
We could also put this stuff in core potentially just to keep the code together

Reply:
Will do. Which `Transaction` should I run with, unless they're supposed to be distinct?
* One: https://github.com/plasma-group/pigi/blob/master/packages/utils/src/interfaces/data-types.ts#L18
* Two: https://github.com/plasma-group/pigi/blob/master/packages/core/src/interfaces/common/utils/state.interface.ts#L17

Reply:
This is a great question. [The first one you listed](https://github.com/plasma-group/pigi/blob/master/packages/utils/src/interfaces/data-types.ts#L18) is the most up-to-date; however, the `method_id` should be optional. As you're starting out with the ownership predicate, just skip the `method_id` because in this case there's only one method, `send()`.

Reply:
As discussed, will make this a larger change in a future PR to merge together `core` and `utils` (tracked in #278)